static/src/js/schemas/uiSchemas/collections/archiveDistributionInformation.js

```diff
@@ -39,6 +45,7 @@ const archiveDistributionInformationUiSchema = {
   ArchiveAndDistributionInformation: {
     'ui:heading-level': 'h4',
     'ui:field': 'layout',
+    'ui:fieldReplacesAnyOrOneOf': true,
```

Comment:
I see a number of places that use anyOf in constraints in the schema, do we need to add this to other places as well?